packages/p2p/src/components/scrollbar-wrapper/scrollbar-wrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useStore } from '@deriv/stores';
+
+type TScrollbarWrapper = {
+    children: JSX.Element;
+    height?: string;
+};
+
+const ScrollbarWrapper = ({ children, height }: TScrollbarWrapper) => {
```

Comment:
what is this component for? is this being used in other pages?

Reply:
I made this component because we got rid of the AutoSizer in my-profile. The reasoning is because now the entire page becomes scrollable (including the p2p pages tab) and the tabs in my-profile dont become scrollable anymore. I wanted to use ThemedScrollbar, but then I didn't want to depend on that component, so I created this component to help with certain components to be scrollable. 

This only happens in my-profile so i could just move it underneath my-profile page folder instead of components, but there are two components where I use the ScrollbarWrapper.